src/popup/components/Modals/ErrorLog.vue

```diff
@@ -14,7 +14,6 @@
       <span>{{ $t('modals.error-log.sub-title') }}</span>
       {{ $t('modals.error-log.content') }}
     </div>
-
```

Comment:
Why removing this space? It's useful to make the code easier to view and analyse.